packages/components/src/components/hds/button-set/index.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+interface HdsButtonSetSignature {
+  Blocks: { default: [] };
+  Element: HTMLDivElement;
+}
+
+export default class HdsButtonSetComponent extends Component<HdsButtonSetSignature> {}
```

Comment:
Just to confirm, did I follow the correct patterns above? @alex-ju

Reply:
no, not really. we would want to use `TemplateOnlyComponent` in this case, as we don't need a backing class.
here's [an example for reference](https://github.com/hashicorp/design-system/blob/f48aeeee8c25a986e7b8529160153c60f6521cc8/packages/components/src/components/hds/accordion/index.ts), I added a suggestion in a separate comment as well.

Reply:
Thanks. Updated to fix